CONTRIBUTOR-ROLES.md

```diff
@@ -0,0 +1,123 @@
+# Contributor Roles
+
+### Triage Team
+Triagers are members that help triage issues. They can set assignments and labels.
+
+* Responsibilities:
+    * Assigning and labeling issues 
+
+* Qualifications: 
+    * Must be a Cilium Organization member
+
+* Privileges:
+    * GitHub Triage role
+
+The process of becoming a Triage Team member is:
+
+1. The contributor is nominated by opening a PR against the member management file under the triage team
+2. At least two members who are already Committers, approve the PR
+
+### Security Team
+Security Team members are listed in the security team file. The security team is responsible for responding to all security issues reported to security@cilium.io or any other community forum. They ensure that the issues are mitigated and a responsible disclosure process is followed in a reasonable time.
```

Comment:
I guess that a subsequent PR will take care of initializing some of these files like the "security team file" and maybe updating the references in these documents so it's clearer where to find them.

Reply:
Yes, I've made an issue for it here to track it https://github.com/cilium/community/issues/4